modules/releasing/pages/adjusting-timeouts-resources.adoc

```diff
@@ -0,0 +1,77 @@
+= Adjusting timeouts and resources
+
+When a Release Pipeline is executed through a PipelineRun, the Pipeline will run until a timeout is reached. This timeout is usually an hour and will make the Release Pipeline to fail if it doesn't finish on time.
```

Comment:
```suggestion
When a Release Pipeline is executed through a PipelineRun, the Pipeline will run until a timeout is reached. This timeout is usually an hour and will make the Release Pipeline fail if it doesn't finish on time.
```

Reply:
English is weird.

You "cause something to fail" and you "make something fail" but you don't "cause something fail" nor do you "make something to fail".

Reply:
Fixed.